### PR TITLE
Update Game.Dat Bravely Default

### PR DESCRIPTION
**Pull request by Ruthalas**
Added a new test case (mine).

### CODE REVIEW

File: games/bravely-default/game.dat [hunk 1]
```diff
@@ -28,3 +28,13 @@ author = "SonofUgly"
...
 gpu = "GeForce GTX 750Ti"
 os = "Windows 8.1"
+
+[[ testcases ]]
+title = "00040000000fc600"
+compatibility = "1"
+date = "2017-11-17"
+version = "HEAD-7ef905e"
+author = "Ruthalas"
+cpu = "i5-4670k @ 4.1GHz"
```
Comment (Hexagon12) [inline games/bravely-default/game.dat]: _Intel i5-4670K..._

File: games/bravely-default/game.dat [hunk 2]
```diff
@@ -28,3 +28,13 @@ author = "SonofUgly"
...
+gpu = "AMD Radeon R9 390"
+os = "Windows 10 x64"
```
Comment (Hexagon12) [inline games/bravely-default/game.dat]: _Windows 10 (your version number)_

File: games/bravely-default/game.dat [hunk 3]
```diff
@@ -28,3 +28,13 @@ author = "SonofUgly"
...
+
+[[ testcases ]]
+title = "00040000000fc600"
+compatibility = "1"
+date = "2017-11-17"
+version = "HEAD-7ef905e"
+author = "Ruthalas"
+cpu = "Intel i5-4670k @ 4.1GHz"
+gpu = "AMD Radeon R9 390"
+os = "Windows 10 Pro"
```
Comment (Leo626) [inline games/bravely-default/game.dat]: It should be the version of Windows 10 you are running. Run command prompt and you will see the version at the top like `Version 10.0.16299` which would be `Windows 10 16299`.

File: games/bravely-default/game.dat [hunk 4]
```diff
@@ -28,3 +28,13 @@ author = "SonofUgly"
 cpu = "i5-6600k @ 4.3GHz"
 gpu = "GeForce GTX 750Ti"
 os = "Windows 8.1"
+
+[[ testcases ]]
+title = "00040000000fc600"
+compatibility = "1"
+date = "2017-11-17"
+version = "HEAD-7ef905e"
```
Comment (Leo626) [inline games/bravely-default/game.dat]: Minor nitpick, this nightly is 10 days old. Testcases can be done with the latest nightly.

File: games/bravely-default/game.dat [hunk 5]
```diff
@@ -28,3 +28,13 @@ author = "SonofUgly"
...
+author = "Ruthalas"
+cpu = "Intel i5-4670k @ 4.1GHz"
```
Comment (TheKoopaKingdom) [inline games/bravely-default/game.dat]: For consistency, `Intel Core i5-4670k` would be better.

**Comment by Leo626** [comment]
Please address the comments or this will be closed.

**Pull request closed without merging**